device/transport/amqp/package.json

```diff
@@ -32,11 +32,11 @@
     "build": "tsc",
     "test-min": "nyc mocha --reporter dot --require source-map-support/register test/_*_test.js",
     "test": "nyc mocha --reporter spec --require source-map-support/register test/_*_test.js",
-    "ci": "npm -s run lint && npm -s run build && npm -s run test-min && npm -s run check-cover",
+    "ci": "npm -s run lint && npm -s run build && npm -s run test-min ",
     "check-cover": "nyc check-coverage --statements 95 --branches 90 --functions 96 --lines 95"
```

Comment:
>check-cover" [](start = 5, length = 12)

We seem to have a check coverage script and coverage values below.  We don't do this other places.